src/store.js

```diff
@@ -36,6 +36,8 @@ export const history = createHashHistory({
   basename: contextPath
 });
 
+history.listen(() => window.scrollTo(0, 0));
```

Comment:
@mogoodrich seems to be a standard issue in react-router. got solution from (https://github.com/ReactTraining/react-router/issues/2019)

Reply:
sounds good!